automated_analysis.py

```diff
@@ -275,6 +301,17 @@ def update_survey_counts(survey_counts, td):
                 themes["Total Relevant Participants"]["Total Participants"] += 1
                 update_survey_counts(themes["Total Relevant Participants"], td)
 
+            set_survey_percentages(themes["Total Relevant Participants"], themes["Total Relevant Participants"])
```

Comment:
I'm probably misunderstanding something, but this seems to initialise everything to 100% - is it needed since you're going to compute the percentage for each code later on anyway?

Reply:
This is only setting the Total Relevant Participants percentages, not each of the individual themes. And the total relevant should be 100%, because all the percentages are calculated out of the total number of relevant. Computing it here and including it in the output is a good sanity check, and reduces the likelihood of misinterpretation because it makes it more obvious what the other percentages are of.

This is a pretty common pattern in this file - handle the total, then handle each individual code/theme in turn, but let me know if there's something I could do here to make that clearer

Reply:
Right, got it, thanks. I don't think you need to change anything, it was just me being unfamiliar with what the output looks like